pages/docs/Argument ${NETWORK_CONFIG}.rst

```diff
@@ -0,0 +1,41 @@
+Argument ${NETWORK_CONFIG}
```

Comment:
All updates looks good to me, but can you please change the filenames of those `rst` to not have any special characters or spaces? These filename seems irregular and might look awkward when appeared in URLs.

Something like `network_config_arguments.rst` or whatnot.